pkg/server/server.go

```diff
@@ -773,8 +807,8 @@ func NewServerWithOpts(opts ...OpenFGAServiceV1Option) (*Server, error) {
 				MaxThreshold:     s.checkDispatchThrottlingMaxThreshold,
 			}),
 			// only create the throttler if the feature is enabled, so that we can clean it afterward
-			graph.WithThrottler(throttler.NewConstantRateThrottler(s.checkDispatchThrottlingFrequency,
```

Comment:
Since there is now two separate resolver, they should not share the same constant rate throttler.  Otherwise, multiple resolver may close the same throttler and causes issue.